packages/plugin-heading-analyzer/package.json

```diff
@@ -25,7 +25,8 @@
     "eslint-config-custom": "*",
     "tsconfig": "*",
     "tsup-config": "*",
-    "typescript": "^4.5.2"
+    "typescript": "^4.5.2",
+    "tsup": "*"
```

Comment:
Thanks @ahmedrowaihi! Can we make this version number explicit, and apply alphabetical sorting?

To match the `core` package:

```
    "tsup": "^6.7.0",
    "tsup-config": "*",
    "typescript": "^4.5.2"
```